src/pages/ProductDetailPage.js

```diff
@@ -0,0 +1,35 @@
+import Card from "../componnets/Card";
+import classes from "./ProductDetailsPage.module.css";
+import { useNavigate, useLoaderData } from "react-router-dom";
+
+const ProductDetailPage = ({ addProductToCart }) => {
```

Comment:
Prop name should be "`onAddToCart`" to follow the common React naming convention for event-like callback props.